media_utils/video_puppeteer.py

```diff
@@ -187,12 +198,18 @@ def playback_done(video):
     # either still playing or there is a problem, so this will never
     # return True.
 
+    active_re = re.compile('NextGetSampleIndex:([\d]+)')
```

Comment:
re variable names: there's no notion of "active" stream in the new MSE, so `active_re` will lead to confusion